pkg/config/reporting.go

```diff
@@ -0,0 +1,77 @@
+package config
+
+// ReportingParams holds reporting parameters
+type ReportingParams struct {
+	Parameters []StepParameters
+}
+
+// ReportingParameters is a global variable with values of reporting parameters
+var ReportingParameters = ReportingParams{
+	Parameters: []StepParameters{
+		{
+			Name:    "gcpJsonKeyFilePath",
+			Aliases: []Alias{{Name: "jsonKeyFilePath"}},
+			ResourceRef: []ResourceReference{
+				{
+					Name:    "gcpJsonKeyFilePathVaultSecretName",
+					Type:    "vaultSecretFile",
+					Default: "gcp",
+				},
+			},
+		},
+		{
+			Name: "gcsFolderPath",
+			ResourceRef: []ResourceReference{
+				{
+					Name:  "commonPipelineEnvironment",
+					Param: "gcsFolderPath",
```

Comment:
Does it need to be a custom property?

```suggestion
					Param: "custom/gcsFolderPath",
```

Reply:
Suggestions have been applied